### PR TITLE
Support for counting idling resource aware rxjava scheduler

### PR DESCRIPTION
**Pull request by athkalia**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -220,7 +220,7 @@ dependencies {
 
     // MVP library
     compile "com.hannesdorfmann.mosby3:mvp:$versions.mosbyMvpVersion"
-    compile "com.hannesdorfmann.mosby3:viewstate:$versions.mosbyMvpVersion"
+    compile "com.hannesdorfmann.mosby3:viewstate:3.0.1" //freaky bug won't work with a non-hardcoded version
```
Comment (miguelslemos) [inline app/build.gradle]: its works for me 
```
// MVP library
    compile "com.hannesdorfmann.mosby3:mvp:$versions.mosbyMvpVersion"
    compile "com.hannesdorfmann.mosby3:viewstate:$versions.mosbyMvpVersion"
    compile "com.hannesdorfmann.mosby3:mvp-nullobject-presenter:$versions.mosbyMvpVersion"
```

File: app/config/gradle/versioning.gradle [hunk 2]
```diff
@@ -23,7 +23,7 @@ ext.versions = [
         espressoTestRulesVersion  : '0.5',
         espressoTestRunnerVersion : '0.5',
         timberVersion             : '4.5.1',
-        mosbyMvpVersion           : '3.0.0',
+        mosbyMvpVersion           : '3.0.1',
         annotationsVersion        : '2.0.1',
```
Comment (miguelslemos) [inline app/config/gradle/versioning.gradle]: a new versions is available 3.0.2

File: app/build.gradle (hunk 1 above)
Comment (athkalia) [reply app/build.gradle]: I tried with your piece of code and it works now. Thanks! I had the exact same thing, something reallly weird must have been happening

File: app/config/gradle/versioning.gradle (hunk 2 above)
Comment (athkalia) [reply app/config/gradle/versioning.gradle]: 👍  updated

**Pull request merged**